### PR TITLE
tree recommendation logic included in library

### PR DESCRIPTION
**Pull request by angsherpa232**

### CODE REVIEW

File: src/recommend.js [hunk 1]
```diff
@@ -37,15 +43,15 @@ function recommend(forestType1, forestType2, future) {
...
+        negative: filterTreeSpecies(three, three2),
       };
     } else {
       result = {
-        one: concatTreeSpecies(one, one2),
-        two: concatTreeSpecies(two, two2),
-        three: concatTreeSpecies(three, three2),
+        positive: concatTreeSpecies([one, one2, two, two2]),
+        neutral: concatTreeSpecies([three, three2]),
+        negative: filterTreeSpecies([one, two, three], [one2, two2, three2]),
```
Comment (friedjoff) [inline src/recommend.js]:
```suggestion
        negative: filterTreeSpecies(concatTreeSpecies(one, two, three), concatTreeSpecies(one2, two2, three2)),
```

File: src/recommend.js [hunk 2]
```diff
@@ -37,15 +43,15 @@ function recommend(forestType1, forestType2, future) {
     );
     if (future) {
       result = {
-        one: filterTreeSpecies(one, one2),
-        two: filterTreeSpecies(two, two2),
-        three: filterTreeSpecies(three, three2),
+        positive: filterTreeSpecies(one, one2),
```
Comment (friedjoff) [inline src/recommend.js]: Please fix the future 😉

Comment (friedjoff) [reply src/recommend.js]: And rename only in the end.

File: src/recommend.test.js [hunk 3]
```diff
@@ -30,16 +30,15 @@ describe('Test for output values', () => {
   test('valid single forest type recommendations', () => {
     expect(recommend('60')).toStrictEqual({
       one: [302800],
-      three: [800, 25400, 60400, 60500, 227200, 363700],
       two: [402300],
+      three: [800, 25400, 60400, 60500, 227200, 363700],
     });
   });
 
   test('valid multiple forest type recommendations', () => {
```
Comment (friedjoff) [inline src/recommend.test.js]: Please also add a test for the future.

File: src/recommend.js (hunk 2 above)
Comment (angsherpa232) [reply src/recommend.js]: Done!

File: src/recommend.js [hunk 4]
```diff
@@ -11,11 +11,13 @@ function recommendTreeSpecies(forestType) {
   return treeSpecies;
 }
 
-const concatTreeSpecies = (treeSp1, treeSp2) =>
-  Array.from(new Set([...treeSp1, ...treeSp2]));
+function concatTreeSpecies(...treeSp) {
+  return Array.from(new Set(treeSp[0].reduce((a, b) => a.concat(b), [])));
```
Comment (friedjoff) [inline src/recommend.js]:
```suggestion
  return Array.from(new Set([treeSp].reduce((a, b) => a.concat(b), [])));
```

File: src/recommend.js [hunk 5]
```diff
@@ -43,13 +45,18 @@ function recommend(forestType1, forestType2, future) {
       };
     } else {
       result = {
-        one: concatTreeSpecies(one, one2),
-        two: concatTreeSpecies(two, two2),
-        three: concatTreeSpecies(three, three2),
+        one: concatTreeSpecies([one, one2, two, two2]),
```
Comment (friedjoff) [inline src/recommend.js]:
```suggestion
        one: concatTreeSpecies(one, one2, two, two2),
```

**Pull request merged**